projects/netgrif-components/src/lib/tabs/tabs.theme.scss

```diff
@@ -10,5 +10,10 @@
             justify-content: flex-end;
         }
     }
+
+    .mat-tab-label-container {
+        border-radius: 6px;
+        background: white;
```

Comment:
is white always ok?

Reply:
i think yes, and if you neeed a specific color for another app you can only define your own style in the app